utilities/pipelines/tokensReplacement/helper/Convert-TokenInFile.ps1

```diff
@@ -5,77 +5,75 @@ Recieves an Input Object of Name/Value and Searches the Current File for the Nam
+Optional. A Boolean That swaps the tokens in the TokenNameValueObject Object. Used to find the Value and Swap with the Name instead. Default is False
 
 .EXAMPLE
 $Object = @(
-    @{ Replace = "TextA"; With = "TextB" }
-    @{ Replace = "TextC"; With = "TextD" }
+    @{ Name = "TextA"; Value = "TextB" }
+    @{ Name = "TextC"; Value = "TextD" }
 )
-Convert-TokensInFileList -Paths 'C:\fileA.txt','C:\fileB.txt' -TokensReplaceWith $Object
+Convert-TokenInFile -FilePath 'C:\fileA.txt','C:\fileB.txt' -TokenNameValueObject $Object
 
 .EXAMPLE
 $Object = @(
-    @{ Replace = "TextA"; With = "TextB" }
-    @{ Replace = "TextC"; With = "TextD" }
+    @{ Name = "TextA"; Value = "TextB" }
+    @{ Name = "TextC"; Value = "TextD" }
 )
-Convert-TokensInFileList -Paths 'C:\fileA.txt','C:\fileB.txt' -TokensReplaceWith $Object -OutputDirectory 'C:\customDirectory'
+Convert-TokenInFile -FilePath 'C:\fileA.txt','C:\fileB.txt' -TokenNameValueObject $Object -OutputDirectory 'C:\customDirectory'
 
 .EXAMPLE
 $Object = @(
-    @{ Replace = "TextA"; With = "TextB" }
-    @{ Replace = "TextC"; With = "TextD" }
+    @{ Name = "TextA"; Value = "TextB" }
+    @{ Name = "TextC"; Value = "TextD" }
 )
-Convert-TokensInFileList -Paths 'C:\fileA.txt','C:\fileB.txt' -TokensReplaceWith $Object -RestoreTokens $true
+Convert-TokenInFile -FilePath 'C:\fileA.txt','C:\fileB.txt' -TokenNameValueObject $Object -SwapValueWithName $true
 #>
-function Convert-TokensInFileList {
+function Convert-TokenInFile {
     [CmdletBinding()]
     param(
         [Parameter(Mandatory)]
-        [string[]] $Paths,
+        [string[]] $FilePath,
 
         [Parameter(Mandatory, ValueFromPipeline = $true)]
-        [psobject] $TokensReplaceWith,
+        [psobject] $TokenNameValueObject,
 
         [Parameter(Mandatory = $false)]
         [string] $OutputDirectory,
 
         [Parameter(Mandatory = $false)]
-        [bool] $RestoreTokens = $false
+        [bool] $SwapValueWithName = $false
     )
-    # Restore Tokens (Swap Replace with Value)
-    if ($RestoreTokens) {
-        Write-Verbose 'Restoring Tokens'
-        $TokensReplaceWith | ForEach-Object {
+    # Swap Value with Name instead
+    if ($SwapValueWithName) {
+        Write-Verbose "Swapping 'Value' with 'Name'"
+        $TokenNameValueObject | ForEach-Object {
             $Name = $PSitem.Value
             $Value = $PSItem.Name
             $PSitem.Name = $Name; $PSitem.Value = $Value
         }
     }
     # Begin the Replace Function
-    Write-Verbose "$($TokensReplaceWith.Count) Tokens Found"
     # Process Path for Token Replacement
-    foreach ($Path in $Paths) {
+    foreach ($Path in $FilePath) {
         # Extract Required Content From the Input
         try {
             $File = Get-Content -Path $Path
-            $FileName = Split-Path $Path -Leaf
+            $FileName = Split-Path -Path $Path -Leaf
         } catch {
             throw $PSItem.Exception.Message
-            exit
+            return
```

Comment:
I think the return is redundant as `throw` will exit the function

Reply:
good pickup. thank you